yarn-project/simulator/src/avm/avm_tree.ts

```diff
@@ -7,6 +7,21 @@ import { type IndexedTreeLeafPreimage, type TreeLeafPreimage } from '@aztec/foun
 import { strict as assert } from 'assert';
 import cloneDeep from 'lodash.clonedeep';
 
+const MAX_TREE_DEPTH = 128;
```

Comment:
I guess this could be a lot smaller - but since it is a linear number of hashes it isnt so bad.